library/core/src/option.rs

```diff
@@ -667,7 +667,7 @@ impl<T> Option<T> {
     /// Returns [`None`] if the option is [`None`], otherwise calls `predicate`
     /// with the wrapped value and returns:
     ///
-    /// - [`Some(t)`] if `predicate` returns `true` (where `t` is the wrapped
+    /// - [`Some`] if `predicate` returns `true` (where `t` is the wrapped
```

Comment:
You've orphaned the mention of `t` here.

Reply:
I think it would be good to remove the brack along with `t` at the back.